config.yaml

```diff
@@ -1,5 +1,6 @@
 common:
  log_file: "/var/log/host-os/builds.log"
+ log_size: 2097152
```

Comment:
If you're adding it here, it can be removed from config.py.
You could actually remove all defaults from there since now everything is in the yaml file.